apps/src/lib/ui/ValidationStep.jsx

```diff
@@ -71,7 +73,12 @@ export default class ValidationStep extends Component {
       <div style={style.root}>
         <div style={{...style.header, ...styleFor(stepStatus)}}>
           <div style={style.icon}>{iconFor(stepStatus)}</div>
-          <div style={style.headerText}>{stepName}</div>
+          <div style={style.headerText}>
+            {stepName}
+            {updatePercentComplete && (
```

Comment:
Since this `ValidationStep` is doing something more general than "updating," I think we should call this prop something like `percentComplete`, `stepPercentComplete`, or `stepProgress.` 


For clarity and handling zero appropriately (assuming you want to show 0%), you could use:
```
const displayPercentComplete = percentComplete >= 0 ? ` ${percentComplete}` : '';

<div style={style.headerText}>
  {`${stepName}{displayPercentComplete}`}
<div>
```

Reply:
Great suggestion! Will do - thanks!